app/features/accessibility.js

```diff
@@ -151,23 +152,23 @@ const render = (el, tip = document.createElement('visbug-ally')) => {
 const determineColorContrast = el => {
   // question: how to know if the current node is actually a black background?
   // question: is there an api for composited values?
-  const text      = getStyle(el, 'color')
+  const color     = getStyle(el, 'fill') || getStyle(el, 'stroke') || getStyle(el, 'color')
```

Comment:
this is the only LOC i'm curious if it's going to have side effects with. like, if an element has a fill color specified, and even though that color is ignored by CSS render engine, visbug tooltip would pick it up and display it yeah? just tested, see attached screenshot.

resolution could be to make the determineColorContrast function node type aware, so it doesnt report a fill on thing not SVG typed. 

![Screen Shot 2019-09-24 at 12 25 29 PM](https://user-images.githubusercontent.com/1134620/65543655-6de31400-dec6-11e9-9e60-03f887d4e739.png)


will play more to see if there's anything else!

Reply:
Yeah right. I've done with checking instance type of element on the new commit.

(BTW does LOC mean 'Level Of Concern'? I didn't know that word 😅)

Reply:
LOC means "line of code", thanks for asking! 👍 

pulled latest on glitch, verified you've fixed the concern!